yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
   - name: gPSFluxMean
     "@id": "#DiaObject.gPSFluxMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Weighted mean of diaSource PSF flux
   - name: gPSFluxMeanErr
     "@id": "#DiaObject.gPSFluxMeanErr"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard error on the weighted mean of diaSource PSF flux
   - name: gPSFluxMin
     "@id": "#DiaObject.gPSFluxMin"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Minimum diaSource PSF flux
   - name: gPSFluxNdata
     "@id": "#DiaObject.gPSFluxNdata"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: The number of data points used to compute gPSFluxChi2
   - name: gPSFluxPercentile05
     "@id": "#DiaObject.gPSFluxPercentile05"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 5th percentile diaSource PSF flux
   - name: gPSFluxPercentile25
     "@id": "#DiaObject.gPSFluxPercentile25"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 10th percentile diaSource PSF flux
   - name: gPSFluxPercentile50
     "@id": "#DiaObject.gPSFluxPercentile50"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Median diaSource PSF flux
   - name: gPSFluxPercentile75
     "@id": "#DiaObject.gPSFluxPercentile75"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 75th percentile diaSource PSF flux
   - name: gPSFluxPercentile95
     "@id": "#DiaObject.gPSFluxPercentile95"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 95th percentile diaSource PSF flux
   - name: gPSFluxSigma
     "@id": "#DiaObject.gPSFluxSigma"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard deviation of the distribution of gPSFlux
   - name: gPSFluxSkew
     "@id": "#DiaObject.gPSFluxSkew"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Skew of diaSource PSF flux
   - name: gPSFluxStetsonJ
     "@id": "#DiaObject.gPSFluxStetsonJ"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: StetsonJ statistic of diaSource PSF flux
   - name: gTOTFluxMean
     "@id": "#DiaObject.gTOTFluxMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Weighted mean of the PSF flux forced photometered at the diaSource position on the calibrated image
   - name: gTOTFluxMeanErr
     "@id": "#DiaObject.gTOTFluxMeanErr"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard error on gTOTFluxMean
   - name: gTOTFluxSigma
     "@id": "#DiaObject.gTOTFluxSigma"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard deviation of the PSF flux forced photometered at the diaSource position on the calibrated image
```

Comment:
"Standard deviation of the individual diaSource TOTFluxMean forced photometered measurements"?